README.md

```diff
@@ -317,17 +227,16 @@ experiment/
     ...
 ```
 
-The `Makefile` (or similar build configuration) should perform any necessary
-compilation or preprocessor steps.
-Each benchmark should expose a function (or method) called `run_iter`
-which is the entry point to the benchmark.
-In the case of compiled languages, it often makes sense to add an extra
-class to an existing benchmark, in order to provide an entry point.
+The `Makefile` should build the VMs and benchmarks needed for the experiment.
```

Comment:
In our experiments we have two Makefiles and the top-level one recursively invokes `make` in the benchmarks dir. It doesn't need to be that way though.